### ISSUE TITLE
Feat: Black (AMOLED) theme

### ISSUE THREAD

**Issue by Daviteusz** [Fox2Code/FoxMagiskModuleManager#217]
Hello,

I personally globally use the black theme (repainter), but FoxMMM resists this unfortunately and lacks the option in it to select black theme.

It would be nice to see a black theme in the app with MaterialYou on or off.

Thanks for your understanding.

Btw. Well done! The app is great!

**Comment by Fox2Code** [comment]
Yeah, I already had this idea in the back of my mind for months now, but I code a bit slow.

Thanks for showing your interest in this feature.
